deployment/operator/Dockerfile

```diff
@@ -3,7 +3,8 @@ ENV VERIFY_CHECKSUM=false
 RUN curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/main/scripts/get-helm-3 \
  && chmod 700 get_helm.sh && ./get_helm.sh
 
-FROM quay.io/operator-framework/ansible-operator:v1.32.0
+# There isn't v1.38.0 image yet.
+FROM quay.io/operator-framework/ansible-operator:v1.36.0
```

Comment:
Is there a .38 image version expected soon ? If yes, should we wait for it, or just file another PR to bump the image version ? If not expected, would it be better to declare our version to be .36 ?